core/base-service/base-graphql.js

```diff
@@ -46,6 +46,9 @@ class BaseGraphqlService extends BaseService {
    *    and custom error messages e.g: `{ 404: 'package not found' }`.
    *    This can be used to extend or override the
    *    [default](https://github.com/badges/shields/blob/master/core/base-service/check-error-response.js#L5)
+   * @param {Function} attrs.transformJson Function which takes the raw json and transforms it before
```

Comment:
Can we specify the default value here so the param is marked as optional when we compile the docs, like this

```suggestion
   * @param {Function} [attrs.transformJson=data => data] Function which takes the raw json and transforms it before
```

(might need to re-run prettier after this for line lengths?)

Reply:
Still new to javascript world :)